src/components/tile-select/tile-select.style.js

```diff
@@ -115,25 +114,23 @@ const StyledTileSelectInput = styled(Input)`
 const StyledTitleContainer = styled.div`
   display: inline-flex;
   align-items: flex-end;
-  margin-bottom: 8px;
+  margin-bottom: 3px;
```

Comment:
Why has this been changed form 8px to 3px?

Reply:
@edleeks87 
 I changed a bit file structure and some css so to have the same "look" as it was before I had to change the margin. Before we had adornments set with position absolute so it didn't count in spacing

Reply:
The "look" has changed anyway though as now the `padding: 24px` is being respected by the TileSelect content (see pic below). IMHO it should remain as 8px as the DS describes how we should implement values as factors of 8 but I'm happy to go with the general consensus here if any other reviewers disagree 

![image](https://user-images.githubusercontent.com/44157880/125925684-6299fd93-160b-46ca-be6d-ace522e66bc9.png)

Reply:
@edleeks87 good spot! I have to double check my values in styles files